ci/check-style.sh

```diff
@@ -31,6 +31,13 @@ find google/cloud -name '*.h' -print0 \
 find google/cloud -name '*.h' -o -name '*.cc' -print0 \
     | xargs -0 sed -i 's/grpc::\([A-Z][A-Z_][A-Z_]*\)/grpc::StatusCode::\1/g'
 
+# Apply cmake_format to all the CMake list files.
+#     https://github.com/cheshirekow/cmake_format
+find . \( -path ./.git -prune -o -path ./third_party -prune \
+          -o -path './cmake-build-*' -o -path ./build-output -prune \) \
```

Comment:
I've only verified this on Linux, but doesn't a `find` action apply to the entire match group? I.e. why couldn't you just apply the `-prune` action once, to the entire first predicate group?

Also, for mental parsing, I'd suggest breaking the predicates up one per line in addition to applying actions to entire groups of predicates; I'm thinking something like:

```
find . \( -path ./.git \
          -o -path ./third_party \
          -o -path './cmake-build-*' \
          -o -path ./build-output \
       \) -prune \
       -o \( -name 'CMakeLists.txt' \
             -o -name '*.cmake' \
          \) -print0 \
    | xargs -0 cmake-format -i
```

Reply:
Also, I *assumed* that everything in that first group was supposed to be pruned -- I noticed the "cmake-build-*" path didn't have `-prune` afterward, but when I ran it, those were pruned anway.